clients/src/main/resources/common/message/ListOffsetsRequest.json

```diff
@@ -34,9 +34,12 @@
   // Version 7 enables listing offsets by max timestamp (KIP-734).
   //
   // Version 8 enables listing offsets by local log start offset (KIP-405).
-  "validVersions": "0-8",
+  //
+  // Version 9 enables listing offsets by last tiered offset (KIP-1005).
+  "validVersions": "0-9",
   "deprecatedVersions": "0",
   "flexibleVersions": "6+",
+  "latestVersionUnstable": true,
```

Comment:
@clolov : Is this change ok with you for 3.9?

@cmccabe : I guess this part won't be included in 3.8?

Reply:
@junrao : right, I'll have a separate PR for 3.8 that will be a bit different (and include the revert)